src/test/java/com/artipie/api/RepositoryRestBaseTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.api;
+
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.asto.fs.FileStorage;
+import io.vertx.junit5.VertxExtension;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Test for {@link RepositoryRest} with `flat` layout.
+ * @since 0.26
+ */
+@ExtendWith(VertxExtension.class)
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public abstract class RepositoryRestBaseTest extends RestApiServerBase {
+    /**
+     * Temp dir.
+     * @checkstyle VisibilityModifierCheck (500 lines)
+     */
+    @TempDir
+    Path temp;
+
+    /**
+     * Test data storage.
+     */
+    BlockingStorage data;
```

Comment:
I'd suggest to make this field private and add getter. If someone changes it from outside, there will be inconsistency, `data` will no longer represent repository data from `repoSettings()` method

Reply:
done

Reply:
@olenagerasimova Please review again